app/controllers/v2/policies_controller.rb

```diff
@@ -54,6 +54,21 @@ def destroy
     permission_for_action :destroy, Rbac::POLICY_DELETE
     permitted_params_for_action :destroy, { id: ID_TYPE }
 
+    def systems
+      hosts_added, hosts_removed = compliance_policy.bulk_assign(
+        V2::PolicyHost, permitted_params[:data]&.dig(:hosts)
+      )
+
+      audit_success(
+        "Updated systems assignment on policy #{compliance_policy.id}, " \
+        "#{hosts_added} added, #{hosts_removed} removed"
+      )
+
+      render_json compliance_policy
+    end
+    permission_for_action :systems, Rbac::POLICY_WRITE
+    permitted_params_for_action :systems, { id: ID_TYPE, data: ParamType.map(hosts: ParamType.array(ID_TYPE)) }
```

Comment:
Is it `hosts` also in the old API? Shouldn't be like:
```suggestion
    permitted_params_for_action :systems, { id: ID_TYPE, data: ParamType.map(systems: ParamType.array(ID_TYPE)) }
```
instead?

Reply:
The v1 implementation used hosts: `relationships: { hosts: { data: [] } }`
I guess it would make sense to use `systems:` in APIv2, so that it corresponds with the name of the endpoint.

Reply:
Yes please

Reply:
I will not yet make this change, as per our PMs.

Reply:
I understand that the discussion resulted in recomendation to use "Systems", rather than "Hosts".

Reply:
Yes, and we need https://github.com/RedHatInsights/compliance-backend/pull/1913 to be merged to unblock us